src/EmailValidatorFactory.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Egulias\EmailValidator;
+
+use Egulias\EmailValidator\EmailValidator;
+use Egulias\EmailValidator\Validation\DNSCheckValidation;
+use Egulias\EmailValidator\Validation\NoRFCWarningsValidation;
+use Egulias\EmailValidator\Validation\RFCValidation;
+
+class EmailValidatorFactory
+{
+    /** @var [] */
+    protected static array $defaultValidators = [
+        RFCValidation::class,
+        NoRFCWarningsValidation::class,
+        DNSCheckValidation::class
+    ];
+    
+    /**
+     * @param string $emailAddress
+     * @return array
+     */
+    public static function create(string $emailAddress): array
```

Comment:
Which are the reasons behing making it static? It prevents being injected as dependency.

Reply:
Also, you are returning the lists of results, not a validator ready for use. I'd suggest to change the name to something like `validationsResults` or similar.